### ISSUE TITLE
Homestead v6.5.0 bad gatweway error 502 trying to use php 5.6

### ISSUE THREAD

**Issue by longestdrive** [laravel/homestead#735]
## Versions
Vagrant: 2.0.1
Provider: VirtualBox - 5.2.2 r119230 (Qt5.6.2)
Homestead: 2.0.1 (using branch v6.5.0)

## Host operating system
Windows 10

## Homestead.yaml
```YAML
ip: "192.168.10.10"
memory: 2048
cpus: 1
provider: virtualbox

authorize: ~/.ssh/id_rsa.pub

keys:
    - ~/.ssh/id_rsa

folders:
    - map: c:/vagrant/projects
      to: /home/vagrant/code

sites:
    - map: ebdowns.local
      to: /home/vagrant/code/ebdowns/ebdowns2014/public

    - map: gcal.loc
      to: /home/vagrant/code/googleapi/public

    - map: wedleague.loc
      to: /home/vagrant/code/wedleague/public

    - map: golfmanager.loc
      to: /home/vagrant/code/golfmanager/golfmanager/public
      php: "5.6"
      
    - map: golfmanager17.loc
      to: /home/vagrant/code/golfmanager17/public

    - map: downscms.loc
      to: /home/vagrant/code/downscms/downscms7/public

    - map: downscmsl5.loc
      to: /home/vagrant/code/downscms2016/public

    - map: spadscms.loc
      to: /home/vagrant/code/spadscms2016.git/public

    - map: spads2016.loc
      to: /home/vagrant/code/spads/public

    - map: mygolfstats.loc
      to: /home/vagrant/code/mygolfstats/public

    - map: mymoney.loc
      to: /home/vagrant/code/mymoney/public

databases:
    - homestead

variables:
    - key: APP_ENV
      value: local
```
## Vagrant up output
[Link to homestead up output](https://gist.github.com/longestdrive/1747339b21b07167be0e2c3a5a6eab40.js)

## Expected behavior
Expecting to be able to run one site using php5.6 with this version

## Actual behavior
Trying to load site creates a 502 Bad Gateway error which per log files relates to missing php5.6-fpm files
Sites using php7 work as expected

This is the error in the nginx log files for the site: 
```connect() to unix:/var/run/php/php5.6-fpm.sock failed (2: No such file or directory) while connecting to upstream```
## Steps to reproduce
1. install latest release of homestead
2. update yaml to add the php5.6 key
3. access site

**Comment by svpernova09** [comment]
Few things going on that may be causing issues:

* You're using Virtualbox 5.2 with a box built on 5.1. We'll be releasing a Virtualbox 5.2 box once PHP 7.2 is officially released
* None of your sites have `php: "5.6"` so they'll all be defaulting to php 7.1, none will be configured to use 5.6.
* I'm not sure what the `php5.6-fpm.sock`  error is referring to, post the results of `systemctl status php5.6-fpm.service` to see what's going wrong.

**Comment by longestdrive** [comment]
Hi - I've updated the yaml file to include the php 5.6 reference. I removed it to ensure I could get the machine up and running and forgot to re-enter it. The problem only occurs when trying to access that site.

I've since deleted the machine again and switched to the master branch of homestead and re-started. The box now works as expected and the site loads correctly. Having got it working I'm reluctant to switch back to v6.5.0 so will close this issue for now but wonder if there's an issue with that release?

Thanks for the help
